tools/loader/ImageClassifier.cpp

```diff
@@ -291,25 +334,41 @@ int main(int argc, char **argv) {
   // if requested from command line.
   loader.compile(ctx);
 
+  assert(!(streamInputFilenamesMode && emittingBundle()) &&
```

Comment:
I'd `GLOW_ASSERT` this, since we'd want this check in Release mode too.

Reply:
Done